tracing-opentelemetry/README.md

```diff
@@ -0,0 +1,70 @@
+# Tracing OpenTelemetry
+
+[![Crates.io: tracing-opentelemetry](https://img.shields.io/crates/v/tracing-opentelemetry.svg)](https://crates.io/crates/tracing-opentelemetry)
+[![Documentation](https://docs.rs/tracing-opentelemetry/badge.svg)](https://docs.rs/tracing-opentelemetry)
+[![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE.txt)
+
```

Comment:
Most of the other crates in the repo have a few more links and badges up here, can we make this consistent with e.g. https://github.com/tokio-rs/tracing/blob/master/tracing/README.md#tracing